packages/google_maps_flutter/google_maps_flutter/example/ios/Podfile

```diff
@@ -31,6 +31,8 @@ target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
   target 'RunnerTests' do
     inherit! :search_paths
+
+    pod 'OCMock'
```

Comment:
This should have a version. See [other uses](https://github.com/flutter/plugins/search?l=Ruby&q=pod+ocmock) for examples.